tests/bson/bug0544.phpt

```diff
@@ -19,121 +19,105 @@ $tests = [
 ];
 
 foreach ($tests as $test) {
-    $bson = MongoDB\BSON\fromPHP($test);
+    $bson = MongoDB\BSON\Document::fromPHP($test);
     /* Note: Although libbson can parse the extended JSON representation for
      * 64-bit integers (i.e. "$numberLong"), it currently prints them as
      * doubles (see: https://jira.mongodb.org/browse/CDRIVER-375). */
-    printf("Test %s\n", MongoDB\BSON\toJSON($bson));
-    hex_dump($bson);
```

Comment:
Was asserting the parsed BSON not relevant to this test?

Reply:
The parsed BSON is printed as part of the debug data of a `MongoDB\BSON\Document` instance. It prints the BSON bytes (base64 encoded for easier handling) as well as the PHP representation, so there's no need to use `hex_dump` as well.

Reply:
Oh, I overlooked the "data" field in the debug output. :+1: